packages/api-request-builder/src/build-query-string.js

```diff
@@ -1,7 +1,7 @@
 /* @flow */
 import type {
   ServiceBuilderDefaultParams,
-} from 'types/sdk'
+} from '../../../types/sdk'
```

Comment:
```types/sdk``` gives this lint error ```unable to resolve path to module import/no-unresolved```, and manually tracing it resolves to the types folder in the root ```../../../types/sdk``` (which gives no errors)